src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -3514,6 +3517,7 @@ def test_sql_tde(self, resource_group, server):
     @ResourceGroupPreparer(location='eastus')
     @SqlServerPreparer(location='eastus')
     @KeyVaultPreparer(location='eastus', name_prefix='sqltdebyok')
+    @live_only() # User tried to log in to a device from a platform (Unknown) that's currently not supported through Conditional Access policy. Supported device platforms are: iOS, Android, Mac, and Windows flavors.
```

Comment:
Did you run this test locally or in some agent? The error msg is rarely seen. I hope this test can be kept as it can pass all previous bumping versions.

Reply:
I am running all tests from Powershell 7.

Reply:
@evelyn-ys any idea for this issue?

Reply:
Which keyvault command did it fail in? I have never met `User tried to log in to a device from a platform (Unknown) that's currently not supported` error before... Why would this test using conditional access policy😂

Reply:
This PR is ready for merge. Do you want me to merge it directly or figure out the root cause of keyvault failure then merge after fix?

Reply:
![image](https://user-images.githubusercontent.com/11583928/195519951-39c53e9f-171a-4f14-b549-ec1622a6f8c0.png)

Reply:
This is in KeyVaultPreparer if I am reading it correctly

Reply:
> This PR is ready for merge. Do you want me to merge it directly or figure out the root cause of keyvault failure then merge after fix?

How do you prefer? I am fine with both, just to make it in the next release